recipes/inferno/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.8" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+
+package:
+    name: inferno
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/inferno-pytorch/inferno
+    git_tag: master
+    # TODO checkout url instead
+    # url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+
+build:
+    number: 0
+    skip: True  # [not py36]
+    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+    build:
+        - python
+        - pip
+    run:
+        - python
+        - pyyaml
+        - scipy
+        - scikit-image
+        - h5py
+        - dill
+        - networkx 1.11
+        - tensorboardx
+        - pytorch
+        - torchvision
+
+test:
+    imports:
+        - inferno
+    # don't run tests for now
+    #source_files:
+    #- tests/test_inferno.py
+    #commands:
+    #- python -m unittest discover -v -s tests
+
+about:
+    license: apache-2.0
```

Comment:
Please add the `` license_file`` and ``license_family``

Reply:
Will do.